src/routes/api/oba/shape/[shapeId]/+server.js

```diff
@@ -1,24 +1,6 @@
-import { error, json } from '@sveltejs/kit';
-import { PUBLIC_OBA_SERVER_URL as baseURL } from '$env/static/public';
-import { PRIVATE_OBA_API_KEY as apiKey } from '$env/static/private';
+import shape from '../../../../../lib/RestAPI/shape.js';
```

Comment:
Check out the `$lib` alias on https://kit.svelte.dev/docs/configuration and use it here to avoid this sort of deep nesting.

Reply:
Actually, I was using the $lib alias, but I noticed that when I used it, the autocomplete and types for oba disappeared. So, I switched to a relative path as a temporary solution.


<img width="748" alt="oba-type" src="https://github.com/user-attachments/assets/c8df1668-cbed-4a2d-88f4-0c25e088f516">



However, after reading the docs, I think we can add a jsconfig.json file to define $lib, so the compiler will recognize it properly.